components/Base/DataTable/index.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div class="datatable">
+    <JdsDataTable
+      local-sort
+      v-bind="$props"
+      v-on="$listeners"
+    >
+      <!-- eslint-disable-next-line vue/valid-v-slot -->
+      <template #item.action="{ item }">
+        <slot name="item.action" :item="item" />
+      </template>
+    </JdsDataTable>
+  </div>
+</template>
+
+<script>
+export default {
+  name: 'BaseDataTable',
+  props: {
+    headers: {
+      type: Array,
+      require: true,
```

Comment:
this props property should be `required` isn't it? @adzharamrullah

Reply:
I think, yes that is typo, should be required not require, i have fixed that by remove because i think i don't needed